includes/class-llms-events.php

```diff
@@ -19,13 +19,18 @@
  * @since 3.37.14 Added `store_tracking_events()` method.
  *                Moved most of the `store_cookie()` method's logic into `store_tracking_events()`.
  * @since 3.37.15 Excluded `page.*` events in order to keep the events table small.
+ * @since [version] Replace singleton code with `LLMS_Trait_Singleton`.
  */
 class LLMS_Events {
 
+	use LLMS_Trait_Singleton;
+
 	/**
-	 * Singleton instance
+	 * Singleton instance.
+	 *
+	 * @deprecated [version] Use {@see LLMS_Trait_Singleton::$instance}.
```

Comment:
Direct reference to the replacement var will not be possible since it's a private class variable. So we should recommend reference to the function rather than the variable.

```suggestion
	 * @deprecated [version] Use {@see self::instance()}.
```

Reply:
Surprisingly, PhpStorm doesn't auto-link to `self::instance()`. How about this?
```suggestion
	 * @deprecated [version] Use {@see LLMS_Trait_Singleton::instance()}.
```

Reply:
:+1: